recipe/conda_build_config.yaml

```diff
@@ -316,7 +318,7 @@ gst_plugins_base:
 gdal:
   - 2.3
 geos:
-  - 3.6.2
+  - 3.7.0
```

Comment:
@pelson and @dopplershift how is cartopy with `geos 3.7.0`?

Reply:
Looks like our test suite runs fine with geos 3.7--so long as Shapely is also built with it.

Reply:
If `cartopy` is OK let's roll this change. (The central pinning and conda-build 3 should ensure that no mismatch of shapely/cartopy/geos would happen.)

Reply:
So, by moving to geos 3.7, we've eliminated Python 2.7 builds on Windows...

Reply:
Could we make the pin conditional?